media-gfx/solvespace/solvespace-2.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DXFRW_COMMIT="87ff1082f49b1286a033ac8f38fe8dde0975bb8b"
+DXFRW_PV="0.6.3"
+DXFRW_PN="libdxfrw"
```

Comment:
Please kill this one and just inline the name; putting it into variable has very little benefit and causes unnecessary indirection.